ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/GroupParametersService.java

```diff
@@ -0,0 +1,17 @@
+package net.corda.v5.ledger.utxo;
+
+import net.corda.v5.membership.GroupParameters;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * {@link GroupParametersService} allows flows to access {@link GroupParameters} for further verification,
+ * for example, accessing to check if issued money is from authoritative party in flows.
+ */
+public interface GroupParametersService {
+
+    /**
+     * Returns the point in time {@link GroupParameters}
+     */
+    @NotNull
+    GroupParameters getGroupParameters();
```

Comment:
I would still call this `getCurrentGroupParameters`, might be worth getting @charlieR3's eyes on this just to make sure he's aware and agrees.

Reply:
Yeah, I agree. `getCurrentGroupParameters` in this context makes more sense I think.